packages/components/src/components/scroller/scroller-styles.ts

```diff
@@ -75,34 +128,62 @@ export const getComponentCss = (
       marginLeft: '-1px', // ensures that the gradient always overlays the content (e.g. when zoomed)
       gridArea: '1 / 1 / 1 / 1',
       justifyContent: 'flex-start',
-      background: `linear-gradient(90deg, ${gradientColor} 50%, ${gradientColorTransparent} 100%)`,
+      background: `linear-gradient(to right, ${getGradient(theme, gradientColorTheme)} 100%)`,
       visibility: isPrevHidden ? 'hidden' : 'visible',
-      '& .button::before': {
-        left: 0,
+      '& .button': {
+        marginLeft: '8px',
+        // Hide buttons on mobile
+        ...hoverMediaQuery({
+          visibility: isPrevHidden ? 'hidden' : 'visible',
+        }),
       },
     },
     'action-next': {
       ...actionPrevNextStyles,
       marginRight: '-1px', // ensures that the gradient always overlays the content (e.g. when zoomed)
       gridArea: '1 / 3 / 1 / 3',
       justifyContent: 'flex-end',
-      background: `linear-gradient(90deg, ${gradientColorTransparent} 0%, ${gradientColor} 50%)`,
+      background: `linear-gradient(to left, ${getGradient(theme, gradientColorTheme)} 100%)`,
       visibility: isNextHidden ? 'hidden' : 'visible',
-      '& .button::before': {
-        right: 0,
+      '& .button': {
+        marginRight: '8px',
+        // Hide buttons on mobile
```

Comment:
Not sure if a hover media query is really a good tool to detect a mobile device.

Reply:
We want to detect it via CSS only. It is also not too dramatic if the arrows are shown as it does not break anything. Do you have a idea on a better way to achieve this?

Reply:
Either use width as we usually do or something like mentioned here: 
https://stackoverflow.com/questions/14942081/detect-if-a-browser-in-a-mobile-device-ios-android-phone-tablet-is-used
or https://stackoverflow.com/questions/12469875/how-to-code-css-media-queries-targeting-all-mobile-devices-and-tablets/42835826#42835826

Reply:
We discussed it with Ole, Stephan and Marcel. We marked it on the long list, for now it should be fine to detect it with the hover mediaquery. It wont break applications.